tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
How is this related?

Reply:
its not, but i'd rather this project be on `esnext` so that things like `foo?.bar` work